packages/react-ui/src/components/molecules/UploadField/UploadField.d.ts

```diff
@@ -1,11 +1,17 @@
 import { TextFieldProps } from '@mui/material/TextField';
+import { InputProps } from '@mui/material';
 
-export type UploadFieldProps = TextFieldProps & {
+export type UploadFieldProps = Omit<TextFieldProps, 'placeholder' | 'onChange'> & {
+  name?: string;
   buttonText?: string;
-  accept?: string[];
+  accept?: string[] | string | null;
   files?: [];
   multiple?: boolean;
-  onChange: (file?: File | null) => void;
+  onChange?: (file?: File | null) => void;
+  inProgress?: boolean;
+  inputProps?: Partial<InputProps>;
```

Comment:
For sake of consistency and understandability i would call these `InputProps` (capitalized) as in mui there is some (big and unexplored ;) difference between `input` and `Input` props ...

Reply:
Ok, changed